doc/man/debos.1

```diff
@@ -1,5 +1,5 @@
 '\" t
-.\" Automatically generated by Pandoc 2.17.1.1
+.\" Automatically generated by Pandoc 3.1.3
```

Comment:
drop this change

Reply:
Should already be gone, since I undid all documentation changes